src/tracer.test.js

```diff
@@ -529,7 +528,7 @@ describe('tracer', () => {
 
   test('No exception at initialization', async done => {
     const mockedTracerGlobals = jest.spyOn(TracerGlobals, 'setHandlerInputs');
-    mockedTracerGlobals.mockImplementation(() => {
+    mockedTracerGlobals.mockImplementationOnce(() => {
```

Comment:
changed because it affected the other tests!
Couldn't find way to write this without mocks :(